collections/concepts/understanding-permissions.md

```diff
@@ -0,0 +1,120 @@
+
+```
+curl "https://api.basistheory.com/tokens" \
+  -H "BT-API-KEY: <BT_API_KEY>" \
+  -H "Content-Type: application/json" \
+  -X "POST" \
+  -d '{
+    "type": "token",
+    "data": "Sensitive Value",
+    "privacy": {
+      "classification": "pii"
+    }
+  }'
+```
+
+## Restriction Policies
+
+In addition to the Impact Level and Classification, Tokens can have a Restriction Policy set. Basis Theory supports two policies: `mask` and `redact`. When the Token is read by an Application with permission to read only lower Impact Levels than the Token, the Restriction Policy defines whether the Token’s data should be masked or redacted. For example, the `social_security_number` Token Type has a default Restriction Policy of `mask` and a default Impact Level of `high`. If an Application that only has permission to read `low` Impact Level Tokens attempts to read a social security number, the data will be returned masked:
+
+```
+curl "https://api.basistheory.com/tokens" \
+  -H "BT-API-KEY: <BT_API_KEY>" \
+  -H "Content-Type: application/json" \
+  -X "POST" \
+  -d '{
+    "type": "social_security_number",
+    "data": "123-45-6789"
+  }'
+```
+
+```
+curl "https://api.basistheory.com/tokens/<TOKEN_ID>" \
+  -H "BT-API-KEY: <BT_API_KEY>" \
+  -H "Content-Type: application/json" \
+  -X "GET"
+
+{
+  "id": "8deb3363-288a-49b2-8c5f-e6b598a8afff",
+  "type": "social_security_number",
+  "data": "XXX-XX-6789",
+  "privacy": {
+    "classification": "pii",
+    "impact_level": "high",
+    "restriction_policy": "mask"
+  },
+  "tenant_id": "c28392d1-15ec-479a-ac15-bbe2764ad66d",
+  "created_by": "b848d641-71fb-439d-808a-efbb8237c282",
+  "created_at": "2022-04-20T23:12:10.401Z"
+}
+```
+
+## Permissions
+
+Each [Application Type](https://docs.basistheory.com/#applications-application-types) has its own set of permissions to choose from (view the [full list here](https://docs.basistheory.com/#permissions-permission-types)).  `Client-side` and `Elements` Applications are only able to `create` Tokens, ensuring that you are able to safely use the Application Key in your client-side code without worrying about a malicious actor using that key to decrypt your data. `Management` Applications can manage infrastructure items only, such as Tenants and Applications, and they cannot interact with Tokens; all other Application Types are restricted to data operations with Tokens. Token permissions come in the format `token:<classification>:<operation>:<scope?>`, for instance `token:bank:read:low` or `token:pci:use:proxy`.
+
+Token Permissions control three things:
+
+1. What Classification of Tokens the Application has access to interact with
+2. What operations the Application can use (create, delete, use, etc.)
+3. How the detokenized plain text data is returned when reading a Token
```

Comment:
This example only applies to `read` permissions. Token permissions always control the first two items, but the third is optional depending on the operation. Today, only `read` and `use` support the `<scope?>` which scopes the operation to an impact level or feature respectively.

Reply:
> when reading a Token

Do you think it needs to be clarified beyond that?